### ISSUE TITLE
Update boot JDK used by CI builds to JDK 10

### ISSUE THREAD

**Issue by kevinrushforth** [javafxports/openjdk-jfx#62]
We've been using JDK 10 as the boot JDK to build FX for a while now, and recently starting using JDK 10 GA (build 46) to build FX 11.

We now plan to make JDK 10 the minimum required JDK. See [JDK-8200446](https://bugs.openjdk.java.net/browse/JDK-8200446) and PR #61 

Before we can do this, the Appveyor and Travis CI builds need to be updated to use JDK 10.

**Comment by kevinrushforth** [comment]
@brcolow do you want to take this issue?

**Comment by brcolow** [comment]
I know that [people are adding](https://github.com/mockito/mockito/blob/release/2.x/.travis.yml#L25) to the Travis CI matrix for installing Java 10 on Travis CI:

```yml
  - install: . ./install-jdk.sh -F 10 -L GPL
    env: JDK='Oracle JDK 10'
```

The second line is just for a label on the Travis UI. To install Java 10 on Appveyor will require adding a build matrix (for Java 9 and 10), un-commenting the code I left behind in our `appveyor.yml`:

```yml
        # $client = New-Object net.webclient
        # $client.DownloadFile("https://download.java.net/java/jdk10/archive/42/BCL/jdk-10-ea+42_windows-x64_bin.exe", "C:\Users\appveyor\jdk-10.exe")
        # Start-Process "C:\Users\appveyor\jdk-10.exe" -ArgumentList "/s ADDLOCAL=`"ToolsFeature,SourceFeature,PublicjreFeature`" /INSTALLDIR=`"C:\jdk-10`" /L `"C:\Users\appveyor\jdk-10-install.txt`""
```

and moving it to only execute for the Java 10 matrix and changing JAVA_HOME for that matrix build.

**Comment by brcolow** [comment]
Working on this in https://github.com/javafxports/openjdk-jfx/pull/63

**Comment by brcolow** [comment]
@kevinrushforth Looks like we will need to also bump the minimum gradle version?

https://ci.appveyor.com/project/javafxports-github-bot/openjdk-jfx/build/develop%20177#L4315

**Comment by kevinrushforth** [comment]
It will build with gradle 4.3 as long as the JDK_HOME variable exactly matches the filesystem path.

**Comment by kevinrushforth** [comment]
> I assume we do not want a Java 9 and 10 build for either Travis or Appveyor? In other words, the matrix should only be JDK 10 macOS and *nix on Travis, and a single build for 10 on Appveyor?

Right. Since I propose to bump the minimum to 10, there would be no need for both.

**Comment by kevinrushforth** [comment]
Fixed by #63
